### PR TITLE
Added some Comparative - Lexical resources

### PR DESCRIPTION
**Pull request by HughP**
I added some comparative lexical resources. I increasingly see this list as multidimensional, the way that Drupal or WordPress can use taxonomies to create sub-lists of data. Is this a direction you want to peruse? I have a URL we can use.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,33 @@ To edit this list, simply [click here](https://github.com/RichardLitt/endangered
 Looking for resources for code languages? Take a look at [the awesome collection of other awesome lists](https://github.com/sindresorhus/awesome).
 
 # Generic Repositories
+_Generic Repositories means what exacty? - that it has a home on sourceforge/github? because not everything in this list is one or the other_
```
Comment (RichardLitt) [inline README.md]: Can you remove this?

File: README.md [hunk 2]
```diff
@@ -144,7 +169,7 @@ _FieldDB_ is actively worked on by the [https://github.com/OpenSourceFieldlingui
 * [UsageDashboard](https://github.com/OpenSourceFieldlinguistics/UsageDashboard) A realtime updating usage dashboard for fielddb servers (forked from dashing) (_Fork_)
 
 ## Modules
-
+_the term "modules" means something in Python. What does it mean here?_
```
Comment (RichardLitt) [inline README.md]: It means something in Javascript, too. I used it here because @cesine did, and she runs the FieldDB project. I assume it means something as part of the FieldDB framework.

Comment (RichardLitt) [inline README.md]: Can you also remove this inline comment? Comments should only be in the issues. Thanks.

File: README.md [hunk 3]
```diff
@@ -296,7 +321,7 @@ Giellatekno does a lot for other minority Uralic languages. Following are some k
...
 * [Projet de recherche sur l'écriture](https://github.com/projetDeRechercheSurLecriture/) crowdsourcing or conducting large scale psycholinguistics experiments (or statistically significant field linguistics)
-* [SIL International](https://github.com/sillsdev) [SIL International](http://www.sil.org/) SIL (originally known as the Summer Institute of Linguistics, Inc.) is probably the leading organization which provides software and tools tailored for use by field linguists and lexicographers working on endangered languages. A little known fact is that it's code is now all open sourced on GitHub and SIL is happy to recieve open source contributions and collaborate on open source projects.
+* [SIL International](https://github.com/sillsdev) [SIL International](http://www.sil.org/) SIL (originally known as the Summer Institute of Linguistics, Inc.) is probably the leading organization which provides software and tools tailored for use by field linguists and lexicographers working on endangered languages. A little known fact is that much of it's code is open sourced on GitHub and SIL is happy to recieve open source contributions and collaborate on open source projects.
```
Comment (RichardLitt) [inline README.md]: This should be in a separate commit, but it's cool.

**Comment by RichardLitt** [comment]
Thanks for adding these. I think subsections should go beneath Software, for now - eventually, we can phase out the giant section if we have good enough list depth, but for now it should be on top. 

I think moving to Drupal or Wordpress is a larger issue that should be talked about in an issue: I've opened https://github.com/RichardLitt/endangered-languages/issues/42 for this.

File: README.md (hunk 3 above)
Comment (HughP) [inline README.md]: Just rephrasing to make it more accurate.

File: README.md (hunk 2 above)
Comment (HughP) [inline README.md]: I'm not sure how to delete the comments after I have already sent the pull request... do I have to edit my copy again and then send a new pull request?

Comment (RichardLitt) [inline README.md]: Basically, on the branch you committed this (master), edit the file to reflect the changes I suggested. Then:

``` sh
git add -A
git commit --amend
``` 
Edit the commit log if you like. :wq should quit if vi is your native editor. Finally: `git push -f origin master`. This will bundle the changes into your previous commit, and force update this PR.

File: README.md (hunk 3 above)
Comment (RichardLitt) [inline README.md]: Yep. Generally commits should reflect one thing at a time - this isn't relevant to the rest of the edits, which means it should be in it's own PR - for instance, I can't accept this without accepting everything else. Just general good practice.

Comment (HughP) [inline README.md]: That is good to know. This is my first pull request. - ever.

Comment (RichardLitt) [inline README.md]: Oh, wow! That's awesome man! I did not know that. Well done. :dancer:

Comment (HughP) [inline README.md]: Just sent an updated pull request.

**Comment by RichardLitt** [comment]
Merged! Thanks.
